db/migrate/20250629190818_add_comment_for_submission_to_team.rb

```
@@ -1,5 +1,7 @@
 class AddCommentForSubmissionToTeam < ActiveRecord::Migration[8.0]
   def change
-    add_column :teams, :comment_for_submission, :string
+    unless column_exists?(:teams, :name)
```

RuboCop (1.86.1)

[convention] Use a guard clause (`return if column_exists?(:teams, :name)`) instead of wrapping the code inside a conditional expression.

(Style/GuardClause)